packages/lesswrong/components/Layout.tsx

```diff
@@ -456,10 +457,11 @@ const Layout = ({currentUser, children, classes}: {
                     <FlashMessages />
                   </ErrorBoundary>
                   <ErrorBoundary>
-                    {showNewUserCompleteProfile
+                    {showNewUserCompleteProfile && !isEAForum
```

Comment:
this can plausibly be FriendlyUI default? or is the content too EA specific 👀?

Reply:
I'd guess it's a bit too specific - all the tags and users to subscribe to are hard-coded (at least for now), and the styles are meant to match the EA login popup which only supports Auth0. It wouldn't be much work to make it more generic in the future, but I don't think it's worth doing without a concrete use case.